### DISCUSSION TITLE
Is there a definite eol for each version?

### DISCUSSION THREAD

**Discussion by MariTwin** [PostgREST/postgrest#2514] [Q&A] [answered]
Hi,

I wonder if there is any statement about end-of-life of each version.
An exact date is preferable but statement about how a version won't be support anymore is enough.

Thanks in advance

**Comment by wolfgangwalther** [comment, marked as answer]
Is that PostgREST version or PostgreSQL version?

PostgREST: We only support the latest version. We are not back-patching anything.

PostgreSQL: We support every PostgreSQL version that has not reached end-of-life upstream, yet. If possible, we support it longer, but there's not strict rule about that, yet.

**Reply by MariTwin** [reply]
Thank you for your answer, I asked about PostgREST version
